context.go

```diff
@@ -119,7 +119,7 @@ func (ctx *Ctx) Accepts(offers ...string) (offer string) {
 	if len(offers) == 0 {
 		return ""
 	}
-	h := ctx.Get(fasthttp.HeaderAccept)
+	h := ctx.Get(HeaderAccept)
```

Comment:
Maybe simplify to:

```go
if h := ctx.Get(HeaderAccept); h == "" {
	return offers[0]
}
```

for more beauty 😉

Reply:
But we use the `h` value later on, if we wrap it inside an `if statement` we lose context of the value and get `h is undefined` :<